pkg/genericapiserver/genericapiserver_test.go

```diff
@@ -115,16 +116,16 @@ func TestInstallAPIGroups(t *testing.T) {
 
 	apiGroupMeta := registered.GroupOrDie(api.GroupName)
 	extensionsGroupMeta := registered.GroupOrDie(extensions.GroupName)
+	s.InstallLegacyAPIGroup("/api", &APIGroupInfo{
```

Comment:
This fails because /api isn't a valid api group prefix as defined on line 109. If you check the returned error, you'll see it's non-nil.